### ISSUE TITLE
macOS - MobileScannerPlugin.swift:93: Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value

### ISSUE THREAD

**Issue by craigbrown-nist** [juliansteenbakker/mobile_scanner#241]
### What
Every so often I get a fatal crash and:
```
mobile_scanner/MobileScannerPlugin.swift:93: Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value
Lost connection to device.
```

**Comment by iklochko** [comment]
Same issue on iOS, current master branch:

```
Passing empty image buffer data.
[api] -[CIRenderDestination initWithBitmapData:width:height:bytesPerRow:format:] requires a valid data argument.
[api] -[CIContext(CIRenderDestination) _startTaskToRender:toDestination:forPrepareRender:forClear:error:] The destination is nil.
mobile_scanner/Util.swift:24: Fatal error: Unexpectedly found nil while unwrapping an Optional value
mobile_scanner/Util.swift:24: Fatal error: Unexpectedly found nil while unwrapping an Optional value
```

**Comment by imtoori** [comment]
this happens all the time on macos if it doesn't detect a qr code

**Comment by zhengcan** [comment]
Please check the PR for this issue: #429

**Comment by juliansteenbakker** [comment]
I have released version 3.0.0 which should fix this issue.
